app/controllers/api/v1/users_controller.rb

```diff
@@ -33,7 +33,9 @@ def index
               city: user[:city],
               training: user[:training],
               experience: user[:experience],
-              gender: user[:gender]
+			        gender: user[:gender],
```

Comment:
The tabbing is strong with this one. :)

We're installing a linter that should automatically catch and fix this stuff, but for now may need to fix it manually.

Reply:
done